package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "zapps",
+  "version": "1.0.0",
+  "description": "Zapps -- Zero-Dependency Applications =====================================",
+  "main": "index.js",
+  "scripts": {
+    "serve": "npx @11ty/eleventy --serve",
+    "build": "npx @11ty/eleventy"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/warpsys/zapps.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
This should be Apache2-or-MIT (or however that's formatted for this system, idk)